src/kakarot/precompiles/precompiles.cairo

```diff
@@ -41,6 +54,24 @@ namespace Precompiles {
     }(evm_address: felt, input_len: felt, input: felt*) -> (
         output_len: felt, output: felt*, gas_used: felt, reverted: felt
     ) {
+        let is_rollup_precompile_ = is_rollup_precompile(evm_address);
+        if (is_rollup_precompile_ != 0) {
+            // Rollup precompiles start at address 0x100
+            tempvar offset = 1 + 3 * (evm_address - FIRST_ROLLUP_PRECOMPILE_ADDRESS);
```

Comment:
just add `+ LAST_ETHEREUM_PRECOMPILE_ADDRESS` to this offset, compute only the `offset` in the `if/else` and keep only one single `jmp` table